src/baseTypes.ts

```diff
@@ -0,0 +1,18 @@
+export enum NetworkId {
```

Comment:
What's the reason for adding this vs using `NETWORKS` in `src/constants.ts`?

Reply:
Ah, I see you've moved this from that file. There's also the `NETWORKS` const, which has additional information to NetworkId. How do we consolidate this?

Reply:
There was a circular dependency which causes the unit test to fail. constants.ts is loading types from other files that need the `NetworkId` enum. Which is why I moved `NetworkId` to a separate `baseTypes.ts` file and imported it in `constants.ts` near the top. Hope that makes sense.

I don't see a `NETWORKS` const. Do you mean `IAddresses`? I don't see any other errors at the moment, but for consistency we can move `IAddresses` to `baseTypes.ts` as well. LMK.

Reply:
ah this makes sense

Reply:
Great catch!

Yes, `IAddresses` is the right one. That's actually the interface. `addresses` is the constant variable. Line 53.

If it makes sense to move it to baseTypes.ts, sure.

Reply:
Done. Please review.